### PR TITLE
Fix readme for AppCode IDE

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**

### CODE REVIEW

File: compose/README.md [hunk 1]
```diff
@@ -15,12 +15,13 @@ git config --global submodule.recurse true
 - Java 11 (should be specified in JAVA_HOME)
 - [macOs/Linux] Android SDK downloaded via `./compose/frameworks/support/jbdeps/android-sdk/downloadAndroidSdk`
 - [Windows] Android SDK downloaded from Android Studio and specified in ANDROID_SDK_ROOT
+- [Optional, MacOS] For AppCode IDE, specify environment variable ANDROID_SDK_ROOT=[YOUR_PATH]/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin (in ~/.zshrc)
```
Comment (pjBooms) [inline compose/README.md:18]: MacOS -> macOs

(to be consistent with above)

Comment (dima-avdeev-jb) [reply compose/README.md]: Done

**Comment by dima-avdeev-jb** [comment]
@pjBooms Look please

**Review by pjBooms — APPROVED** [review]

File: compose/README.md [hunk 2]
```diff
@@ -15,12 +15,13 @@ git config --global submodule.recurse true
 - Java 11 (should be specified in JAVA_HOME)
 - [macOs/Linux] Android SDK downloaded via `./compose/frameworks/support/jbdeps/android-sdk/downloadAndroidSdk`
 - [Windows] Android SDK downloaded from Android Studio and specified in ANDROID_SDK_ROOT
+- [Optional, macOs] For AppCode IDE, specify environment variable ANDROID_SDK_ROOT=[YOUR_PATH]/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin (in ~/.zshrc)
```
Comment (igordmn) [inline compose/README.md:18]: Let me check if it requires for IDEA too

Comment (dima-avdeev-jb) [reply compose/README.md:18]: Can I merge?

Comment (igordmn) [reply compose/README.md:18]: In IDEA, SDK should be read automatically.

IDEA (or more specifically Android Plugin for IDEA platform, that should also be the same in AppCode) reads SDK from local.properties, and `androidx` project writes SDK into it via SdkHelper.

I am not 100% sure, that we shouldn't specify SDK for AppCode, but if possible, better to not add different instructions for each IDE (or extra instructions that may not needed).

Could you check, what happen, if you:
1. remove Android SDK in the settings (or point to another the location)
2. close AppCode
3. remove local.properties file
4. open AppCode again.

Will Android SDK point to `jbdeps/android-sdk/darwin`?

Comment (dima-avdeev-jb) [reply compose/README.md:18]: > In IDEA, SDK should be read automatically.
> 
> IDEA (or more specifically Android Plugin for IDEA platform, that should also be the same in AppCode) reads SDK from local.properties, and `androidx` project writes SDK into it via SdkHelper.kt.
> 
> I am not 100% sure, that we shouldn't specify SDK for AppCode, but if possible, better to not add different instructions for each IDE (or extra instructions that may not be needed).
> 
> Could you check, what happen, if you:
> 
>     1. remove Android SDK in the settings (or point to another the location)
> 
>     2. close AppCode
> 
>     3. remove local.properties file
> 
>     4. open AppCode again.
> 
> 
> Will Android SDK point to `jbdeps/android-sdk/darwin`?

On my Mac it works good and successfully import project. 
After it in local.properties automatically adds this lines:
```
sdk.dir=/Users/dim/Desktop/github/JetBrains/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin
cmake.dir=/Users/dim/Desktop/github/JetBrains/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin/native-build-tools
```

@pjBooms Could you test this case on your Mac when you have time?

**Pull request merged**